src/Misc/layoutbin/en-US/strings.json

```diff
@@ -124,69 +124,60 @@
     "Visual Studio Team Services Agent",
     "Copyright (c) Microsoft Corporation",
     "",
-    "Run from the directory above the bin agent folder to allow for updates",
+    "Run the agent through the convenience cmd or shell script wrappers.",
     "",
-    "Alternatively, you can run the convenience cmd or shell script wrappers.",
-    "Arguments will be passed along",
+    "Print agent version information:",
+    "Win  : config.cmd --version and config.cmd --commit",
```

Comment:
We should prefix with `.\` for windows instructions since ps requires it and it works in cmd-prompt too.